### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mecevit**
Added Tutorial, Bug Reporting, Contributing Code and Learn More sections to the main README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ limitations under the License.
 <p align="center">
     <br>
     <a href="https://layer.ai">
-        <img src="https://app.layer.ai/assets/logo.svg" width="200" alt="Layer"/>
+          <img src=".github/assets/layer_logo_light.png" width="512" alt="Layer"/>
```
Comment (dpp23) [inline README.md:20]: This readme is uploaded to https://pypi.org/project/layer/ so these images won't load there if they are loaded from local files.

**Review by cihati — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -39,12 +39,64 @@ limitations under the License.
...
+
+train()
+```
+
+Now you can fetch your model from Layer:
+
+```python
+import layer
+
+clf = layer.get_model("my-model:1.1")
```
Comment (kocabiyikalper) [inline README.md:82]: this should be `layer.get_model("my-model:1.1").get_train()` to return `SVC()`.

**Review by kocabiyikalper — APPROVED** [review]
LGTM. One small comment

File: README.md (hunk 2 above)
Comment (mecevit) [reply README.md]: Good catch. Fixed

**Review by volkangurel — APPROVED** [review]

**Pull request merged**